src/test/java/com/nurkiewicz/java8/J05_MultiRunnerTest.java

```diff
@@ -51,7 +49,7 @@ public void shouldRunMultipleBlocks() throws Exception {
 		));
 
 		//then
-		await().untilCall(to(counter).sum(), is(1L + 2L + 3L));
+		await().until(() -> assertThat(counter.sum()).isEqualTo(1L + 2L + 3L));
```

Comment:
In addition from my test related trainings I know that `to` proxy (here or in old catch-exception) is not obvious to everyone from the beginning.